appinfo/routes.php

```diff
@@ -60,6 +60,8 @@
 			'verb' => 'POST', 'requirements' => ['collectiveId' => '\d+', 'parentId' => '\d+']],
 		['name' => 'page#touch', 'url' => '/_api/{collectiveId}/_pages/{id}/touch',
 			'verb' => 'GET', 'requirements' => ['collectiveId' => '\d+', 'id' => '\d+']],
+		['name' => 'page#setFullWidth', 'url' => '/_api/{collectiveId}/_pages/{id}/fullWidth',
```

Comment:
Mates, WDYT about creating some generic "update" method to store small things like Full width/Emoji/etc?

Reply:
Should I remove old api endpoint completely or just deprecate it?

Reply:
I've created separate issue for that https://github.com/nextcloud/collectives/issues/1415. Better to not do everything in single PR.